src/Medidata.ZipkinTracer.Core.Collector/Properties/AssemblyInfo.cs

```diff
@@ -37,7 +37,7 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.3.0")]
-[assembly: AssemblyFileVersion("1.3.0")]
-[assembly: AssemblyInformationalVersion("1.3.0")]
+[assembly: AssemblyVersion("1.4.0")]
```

Comment:
maybe we need to have a share assembly file for whole solution instead for each project. Or at least remove Assembly version information in this file. What do you think? @kenyamat @lschreck-mdsol